### ISSUE TITLE
[ta-rft] missing title

### ISSUE THREAD

**Issue by jVanZwieten** [Infragistics/convertwire#231]
//reproduce
version-free-assembly-names.adoc
http://help.infragistics.com/doc/TestAutomationIBMWinForms/2015.2?page=Version-Free_Assembly_Names.html

//result
Title (Version-Free Assembly Names) is missing.

**Comment by ig-ealbert** [comment]
This is because there is no topic title or contents title defined in the Document X file:

```
<Topic Id="b1a5bc0c-44a9-4912-8a1b-7a578260d1ca" Status="New Topic" CreatedOn="2010-11-08T16:13:23.7446225Z" ModifiedOn="2013-11-06T18:50:18.16967Z" PageTypeName="" AutoIndex="true" HelpContextIds="" Name="Version-Free_Assembly_Names" BuildFlags="">
  <Title />
  <ContentsTitle />
```

This should be fixed in Document X.  Log a bug in TFS.

**Comment by ig-ealbert** [comment]
Fixed in #250?

**Comment by craigshoemaker** [comment]
yes, 52f32f20f85222a63d5096e89bee4db2abbccfde closed this issue
